src/library_async.js

```diff
@@ -110,7 +110,7 @@ mergeInto(LibraryManager.library, {
       dbg('asyncify instrumenting exports');
 #endif
 #if ASYNCIFY == 2
-      var ASYNCIFY_EXPORTS = {{{ JSON.stringify(ASYNCIFY_EXPORTS) }}};
+      var exportPatterns = [{{{ ASYNCIFY_EXPORTS.map(x => new RegExp('^' + x.replace(new RegExp('\\*', 'g'), '.*') + '$')) }}}];
```

Comment:
I don't think you need the outer `new RegExp`.. you can just use `/` at the beginning and end, no? Like in the other locations where we do this?